inc/admin/diagnostics/namespace.php

```diff
@@ -134,7 +134,6 @@ function render_page() {
 	$output .= 'MySQL Version: ' . $wpdb->db_version() . "\n";
 	$output .= 'Webserver Info: ' . $_SERVER['SERVER_SOFTWARE'] . "\n\n";
 	$output .= "#### PHP Configuration\n\n";
-	$output .= 'Safe Mode: ' . ( ini_get( 'safe_mode' ) ? 'Enabled' : 'Disabled' . "\n" );
```

Comment:
Safe Mode was removed from PHP 5.4 (http://php.adamharvey.name/manual/en/features.safe-mode.php). Since we require PHP >7.3 to run recent releases of Pressbooks, we can probably remove this from the diagnostics page.